HorizonEngine/src/HorizonEngine/AssetManagement/AssetManager.h

```diff
@@ -6,27 +6,35 @@
 #include "HorizonEngine.h"
 #include "pch.h"
 #include <Renderer/Texture.h>
+#include <Renderer/Sprite.h>
 
 namespace Hzn
 {
 	class HZN_API AssetManager
 	{
 	public:
 		AssetManager() {};
-		~AssetManager() {};
+		~AssetManager() {
```

Comment:
Why don't we clear audio along within the destructer?

Reply:
I think it's needed.  I was ignoring it because we're not using audio for a while, my bad. Anyway, Today I will optimize the asset manager

Reply:
No worries, that's understandable. Great Work!